src/components/form/Form.tsx

```diff
@@ -0,0 +1,78 @@
+import React, {useState} from "react";
+import classNames from "classnames";
+
+import {ApiErrorShape} from "../../core/network-manager/networkModels";
+import {generateErrorMessage} from "../../utils/error/errorUtils";
+import {ArrayToUnion} from "../../utils/typeUtils";
+import {ERROR_TYPES} from "../../utils/error/errorConstants";
+
+export interface FormProps {
+  testid: string;
+  onSubmit: () => void;
+  children: React.ReactNode;
+  errorInfo: ApiErrorShape | null;
+  knownErrorKeys: string[] | null;
+  onChange?: (event: React.SyntheticEvent) => void;
+  customClassName?: string;
+  isSubmitAllowed?: boolean;
+  shouldDisplayFieldErrorCountInErrorMessage?: boolean;
+  skipErrorTypes?: Array<ArrayToUnion<typeof ERROR_TYPES>>;
+}
+
+function Form(props: FormProps) {
+  const {
+    children,
+    testid,
+    onSubmit,
+    customClassName,
+    errorInfo,
+    knownErrorKeys,
+    isSubmitAllowed = true,
+    skipErrorTypes
+  } = props;
+  const [isPristine, setPristine] = useState(true);
+  const containerClassName = classNames("form", customClassName);
+  const formError = generateErrorMessage(errorInfo, {
+    knownErrorKeys,
+    skipTypes: skipErrorTypes
+  });
+
+  return (
+    <form
```

Comment:
Maybe we can add `autocomplete` attribute with default value set to `"off"`. I needed it a couple of times but I'm not sure if we need to add this to project starter

Reply:
I think we can add it as a prop and it can default to `"off"`. What do you think ?

Reply:
I agree